lib/wos_record.rb

```diff
@@ -0,0 +1,160 @@
+require 'htmlentities'
+
+# Utilities for working with a Web of Science record
+class WosRecord
+
+  # @return doc [Nokogiri::XML::Document] WOS record document
+  attr_reader :doc
+
+  # @param record [String] record in XML
+  # @param encoded_record [String] record in HTML encoding
+  def initialize(record: nil, encoded_record: nil)
```

Comment:
How about a some protection, like:
```ruby
raise ArgumentError, 'Either record or encoded_record parameter is required' if record.nil? && encoded_Record.nil?
```

Reply:
Yes, for sure.  It's already in there, see the private method.  It's OK if `record` is nil, but when it is, the `encoded_record` cannot be nil.  The specs check that this raises an exception (IIRC).

Reply:
See https://github.com/sul-dlss/sul_pub/pull/275/files#diff-3062786315e5c2629414d0c5972f1caaR150